README.md

```diff
@@ -21,6 +21,7 @@ YCM-Generator can also be used as a Vim plugin. Once installed with Vundle/NeoBu
 * Supported build systems:
     + Make
     + CMake
+    + qmake
```

Comment:
I'd prefer consistency over accuracy here, since the inconsistency looks ugly. Realistically, having them all being lowercase or all being capitalized are equally good, so I don't really mind which of those two you use. (And all being lowercase is probably more grammatically correct anyway)